README.md

```
@@ -69,6 +69,40 @@ swift run ExportOtherReminders /path/to/vault --cleanup
+
+```bash
+# Clean vault for fresh sync
+swift run ReSyncReminders /path/to/vault
+
+# After running, use RemindersSync to complete the fresh sync
+swift run RemindersSync /path/to/vault
+```
+
+### 5. CleanUp
+
+**Remove only completed tasks while preserving incomplete ones**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)